quince/plugins.yml

```diff
@@ -12,7 +12,7 @@
     using an external service based on the codejail library.
 
 - name: mfe_extensions
-  src: '-e git+https://github.com/eduNEXT/tutor-contrib-mfe-extensions.git@hpg/quince-support#egg=tutor-contrib-mfe-extensions==v17.0.0'
+  src: '-e git+https://github.com/eduNEXT/tutor-contrib-mfe-extensions.git@hpg/quince-support'
```

Comment:
hi @mariajgrimaldi, 

I tried to install the URL but it tells me that the branch `hpg/quince-support` does not exist

Reply:
I'll remove the branch reference altogether then. Thanks!

Reply:
I think it's better if we use the tag `v17.0.0` in that way we avoid updates in the future when master passes to be the next release